lhotse/cut/multi.py

```diff
@@ -0,0 +1,405 @@
+                    ),
+                    custom={
+                        k: merge_custom(
+                            k,
+                            (
+                                s.custom[k]
+                                for s in csups
+                                if s.custom is not None and k in s.custom
+                            ),
+                        )
+                        for k in custom_keys
+                    },
+                    alignment={
+                        # Concatenate the lists of alignment units.
+                        k: reduce(
+                            add,
+                            (
+                                s.alignment[k]
+                                for s in csups
+                                if s.alignment is not None and k in s.alignment
+                            ),
+                        )
+                        for k in alignment_keys
+                    },
+                )
+            )
+
+        return fastcopy(self, supervisions=msups)
+
+    @staticmethod
+    def from_mono(*cuts: DataCut) -> "MultiCut":
+        """
+        Convert one or more MonoCut to a MultiCut. If multiple mono cuts are provided, they
+        must match in all fields except the channel. Each cut must have a distinct channel.
+
+        :param cuts: the input cut(s).
+        :return: a MultiCut with a single track.
+        """
+        from .mono import MonoCut
+
+        assert all(isinstance(c, MonoCut) for c in cuts), "All cuts must be MonoCuts"
+        assert (
+            sum(
+                1 for _ in groupby(cuts, key=lambda c: (c.recording_id, c.start, c.end))
+            )
+            == 1
+        ), "Cuts must match in all fields except channel"
+        assert len(set(c.channel for c in cuts)) == len(
+            cuts
+        ), "All cuts must have a distinct channel"
+
+        data = cuts[0].to_dict()
+        data.pop("type")
+
+        return MultiCut(
+            **{
+                **data,
+                "channel": sorted([c.channel for c in cuts]),
+                "supervisions": [s for c in cuts for s in c.supervisions],
+            }
+        )
+
+    def to_mono(self) -> List["DataCut"]:
+        """
+        Convert a MultiCut to a list of MonoCuts, one for each channel.
+        """
+        from .mono import MonoCut
+
+        return [
+            MonoCut(
+                id=f"{self.id}-{channel}",
+                recording=self.recording,
+                start=self.start,
+                duration=self.duration,
+                channel=channel,
+                supervisions=[
+                    fastcopy(s, channel=channel)
+                    for s in self.supervisions
+                    if channel in s.channel
+                ],
+            )
+            for channel in self.channel
+        ]
+
+    @staticmethod
+    def from_dict(data: dict) -> "MultiCut":
```

Comment:
It looks like serialization of MultiCut is missing test coverage, can you pls add some tests for this?

Reply:
Done.